generator/resources/ec2_userdata_test_matrix.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "os": "ol9",
+        "username": "ec2-user",
+        "instanceType": "t3a.medium",
+        "installAgentCommand": "go run ./install/install_agent.go rpm",
```

Comment:
How will this work? We're also installing the agent directly in user data

Reply:
Originally this line does not actually get executed since we are hard-coding the install command. However I made new change to run the install_agent.go script to install therefore it should address this comment.

Reply:
This command gets executed in the terraform after the instance is created but before the integration test. Take a look at `terraform/ec2/linux/main.tf`. What you have, you will launch the agent via user data, and then the test will go and replace it. This value needs to be changed. It should not run installation here.

Reply:
I was very confused about this to but I think here, the same var is being passed to the shell script [here](https://github.com/aws/amazon-cloudwatch-agent-test/pull/219/files#diff-21f80332df902cb88dd8d816f1fb3c758d8884f0a30b2054a44ec3c91044bb1bR39) and the remote-exec part to install agent via terraform after init/before integ test is removed.
So should be fine *I think* - but confusing.